### ISSUE TITLE
Comparison with openai CLIP on multilingual data

### ISSUE THREAD

**Issue by milmin** [UKPLab/sentence-transformers#1306]
Hi, this is more of a question / discussion rather than an issue.
I have run a test doing inference with native openai clip (a priori trained only on english texts if I'm not mistaken) and your multilingual sbert clip on the same data made by fashion objects with their caption in french. Interestingly and strangely enough, I noticed a better performance of multilingual sbert on zero-shot predictions on images but not on texts. 

In more details, I encoded on one side images with `SentenceTransformer('clip-ViT-B-32')`, captions and a set of labels for image classification (like "shoes", "t-shirt", "skirt", etc, all in french) with `SentenceTransformer('sentence-transformers/clip-ViT-B-32-multilingual-v1')`. On the other side I did the same with openai clip. 
After that I computed similarities between encoded images and encoded labels (like `image_features @ labels_features.T`) and between encoded captions and encoded labels. I extracted the top1 and top5 most similar labels and I computed accuracy.

Results show that 
- top1 and top5 accuracy on images are better with sbert clip multilingual
- openai clip performs much better on captions although it was trained on english

Did you notice something similar? How could you explain this result?

**Comment by nreimers** [comment]
This is quite interesting.

I notice that you use dot-product (image_features @ labels_features.T), but as far as I know, CLIP uses cosine similarity. So you might need to normalize your vectors before you compute the dot product.

**Comment by milmin** [comment]
Yes sure, the encodings are normalized before computing the dot product
